python/remotivelabs-broker/remotivelabs/broker/sync/client.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3
+
+import binascii
+import json
+import queue
+import sys
+from threading import Thread
+from typing import Union, Callable, List, Iterable
+
+from . import SignalCreator
+from . import helper as br
+from ..generated.sync import network_api_pb2 as network_api
+from ..generated.sync import network_api_pb2_grpc
+from ..generated.sync import system_api_pb2_grpc
+from ..generated.sync import traffic_api_pb2_grpc
+
+
+class SignalWrapper:
```

Comment:
Why not simply call it signal? I can buy it if it is to be explicit in demo purpose or similar but otherwise I believe that just Signal is fine since the context is inherited from broker/sync. What do you think?

Reply:
Lets talk about this moving on

Reply:
`SignalValue`? `SignalSample`.

Reply:
> `SignalValue`? `SignalSample`.

Good proposals, we use SignalValue in javascript so perhaps thats a good name.

Reply:
I renamed this to SignalValue for now